readme.md

```diff
@@ -44,14 +44,14 @@ func (c *Counter) HandleEvent(ctx context.Context, e *live.Event) error {
     return nil
```

Comment:
readme also needs the name change to `RenderLayout` fixed, maybe its contents as well?